### ISSUE TITLE
email list link

### ISSUE THREAD

**Issue by erikao** [mozilla/mozilla-opennews#195]
relates to issue #194 

setting aside the email strategy conversation, are we pointing people to two separate lists for separate purposes (mailchimp: https://source.opennews.org/en-US/subscribe/ and mojo: https://lists.mozilla.org/listinfo/community-mojo)?

if yes, we should audit email links to make sure they refer people to the appropriate list for the purpose.
if no, we should decide which is the single list, and update all references to that list.

**Comment by sinker** [comment]
So this feels like an existential question: what do we do with two lists we barely use?

**Comment by erikao** [comment]
yes, if it's possible to have just a segment of the needed bigger conversation, let's make that decision.

if it's not possible to do that, then continuing status quo is not a problem. it just feels like if we launch with a wee bit of fanfare, we might see at least a small flurry of subscriptions so if we had some consistency, that'd be good.

basically -- mojo is a totally open list, anyone can email into it, it could become a discussion list, though the discussion portion of it is VERY light (like, maybe once a month). it's basically an info blast list of ours for right now--community calls, applications for things, etc. BUT it has been around since the start of the program and somehow sometimes people read it (like Josh Benton emailed me off of it about something totally unrelated).

the source list is (I think) what we used for SRCCON and is through mailchimp and would be the basis for a more traditional enewsletter communication. if we wanted, we could probably encourage the mojo people to move over to this list, once we have a plan and publication schedule set for it. it feels less likely that we could migrate people from this list to the mojo list since the mojo list is very mozilla-y and not as familiar to many readers (my guess).

so basically-- (once we develop an actual email strategy) do we expect to use these two lists differently and want to continue directing people to separate lists OR is the more realistic move going to be to consolidate them?

**Comment by sinker** [comment]
hmmmmm. 

I think we status-quo for now tbh.

**Comment by erikao** [comment]
yep, totally fine. just raising it in case there was a quick and easy answer.

**Comment by sinker** [comment]
yeah, closing for now.
